### ISSUE TITLE
[SF_ACC][FileUploader]: The file uploader without input field cannot work using keyboard when the screen reader is on

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents#10656]
### Bug Description

The file uploader without input field cannot work using keyboard, neither Enter nor Space, when JAWS is on.

### Isolated Example

https://sap.github.io/ui5-webcomponents/nightly/components/FileUploader/#button-only

### Steps to Reproduce

1. Navigate to https://sap.github.io/ui5-webcomponents/nightly/components/FileUploader/#button-only 
2. Open JAWS2024
3. Focus the Upload button
4. Press Enter key

### Priority

None

### UI5 Web Components Version

2.7.0-rc.1

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by didip1000** [comment]
Hey @SAP/ui5-webcomponents-topic-b,

This one's for us. It could be similar to issue #3767. The issue can be replicated when the virtual cursor is disabled.

Best,
Diana
